test/parallel/test-http-mutable-headers.js

```diff
@@ -79,8 +79,7 @@ const s = http.createServer(common.mustCall((req, res) => {
         'set-cookie': cookies,
         'x-test-array-header': arrayValues
       });
-      // eslint-disable-next-line no-restricted-properties
-      assert.deepEqual(headersCopy['set-cookie'], cookies);
+      assert.deepStrictEqual(headersCopy['set-cookie'], cookies);
```

Comment:
Re: https://github.com/nodejs/node/issues/12885#issuecomment-299725052
IMHO since this is intentional we need to add
```assert.strictEqual(Object.getPrototypeOf(headersCopy), null);```